packages/react/src/components/InAppMessaging/BannerMessage/styles.css

```diff
@@ -0,0 +1,84 @@
+.body {
+  color: #000;
```

Comment:
We need to place all styling inside ui package where we use tokens in style dictionary to maintain CSS for every components. I am happy to give you a walkthrough if you want :)